src/toil/test/jobStores/jobStoreTest.py

```diff
@@ -846,6 +845,20 @@ def _cleanUpExternalStore(url):
         blobService, containerName, _ = AzureJobStore._extractBlobInfoFromUrl(urlparse.urlparse(url))
         blobService.delete_container(containerName)
 
+@needs_azure
```

Comment:
Are the addition of these tests sufficient for testing the issues being resolved?